site/src/App.jsx

```diff
@@ -7,12 +7,14 @@ import { useState, useEffect, useRef } from "react";
 import Tour from "./Tour";
 import StartupBox from "./StartupBox";
 import { ThemeProvider } from "@mui/material";
+import Sidecar from "./Sidecar";
 
 function App() {
   const [modeName, setModeName] = useState(getTheme());
   const [run, setRun] = useState(false);
   const [tour, setTour] = useState(!(localStorage.getItem("tour") === "true"));
   const [open, setOpen] = useState(tour);
+  const [sidecarOpen, setSidecarOpen] = useState(!open);
```

Comment:
the term 'sidecar' already has a meaning in GIS/Mapping- it means 'an addon or additional dataset'. 

We should change the title of this to something like 'navigator panel' because it's job is to jump the user from place to place.